### PR TITLE
MMDP-410 Implement adding state dropdown

### PR DESCRIPTION
**Pull request by p8ul**
**What does this PR do?**

- Allow users to manage options in the 'State' drop down.

**description of Task to be completed?**

- Implement create new state dropdown
- Implement update state dropdowns
- Implement delete state dropdowns
- Implement viewing the option in the 'State' drop-down field on the Stakeholder 'Basic Information' and 'Beneficiary Services' forms respectively

**How should this be manually tested?**

- Fetch and checkout to ft-create-state-MMDP-410
- On your browser navigate to /dropdowns/state from the CMS

**Any background context you want to provide?**

- None

**What are the relevant Jira issues?**

[MMDP-410](https://viisaus.atlassian.net/browse/MMDP-410)

**Screenshots (if appropriate)**

Screenshot 2019-03-19 at 21 52 10

Screenshot 2019-03-19 at 22 03 09

**Questions:**

- None

### CODE REVIEW

File: client/assets/styles/pages/_state.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    &__actions {
+        color: white;
+        background: $cool-blue;
+        border-radius: 50%;
+        height: 34px;
+        width: 107px;
+        margin-top: 36px;
+        opacity: 1;
+        &:hover {
```
Comment (bozicschucky) [inline client/assets/styles/pages/_state.scss]: @p8ul  the hover doesn't happen to be working.

File: client/components/common/Form/DropdownForm.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        value={item.description}
+        name="description"
+      />
+      <div className="stateDropdown__actions">
+        <ActionModal
+          confirmDeleteGroup={deleteAState}
+          group={item}
+          header="Delete a state"
+          content="Confirm delete state"
+          triggerText={<Icon name="trash alternate outline" size="large" />}
```
Comment (bozicschucky) [inline client/components/common/Form/DropdownForm.js]: The model returns an invalid prop type in the console.
please pass the right prop type to this action modal

**Comment by bozicschucky** [comment]
Screenshot 2019-03-20 at 08 24 20

@p8ul the css is not working well on 100% zoom of the page.

**Comment by bozicschucky** [comment]
Screenshot 2019-03-20 at 08 29 22
On posting successfully the drop downs, two sucess toasters are shown.

File: client/assets/styles/pages/_state.scss (hunk 1 above)
Comment (p8ul) [reply client/assets/styles/pages/_state.scss]: Thank you @bozicschucky for the feedback. From the mockup clarification. The delete button should be visible regardless of the hover events. But I have removed the hover CSS code.

**Pull request merged**